flink-filesystems/flink-hadoop-fs/src/test/java/org/apache/flink/runtime/fs/hdfs/HadoopFsFactoryTest.java

```diff
@@ -61,4 +72,61 @@ public void testCreateHadoopFsWithMissingAuthority() throws Exception {
 			assertTrue(e.getMessage().contains("authority"));
 		}
 	}
+
+	@Test
+	public void testReadBufferSize() throws Exception {
```

Comment:
I'd probably divide the test into different cases: One without setting property, one with setting it to 0, one with setting it to >0. The additional lines should make the tests much easier to read.